core/server/web/shared/middlewares/validation/profile-image.js

```diff
@@ -0,0 +1,21 @@
+const common = require('../../../../lib/common');
+const imageLib = require('../../../../lib/image');
+
+module.exports = function profileImage(req, res, next) {
+    // we checked for a valid image file, now we need to do validations for profile image
+    imageLib.imageSize.getImageSizeFromPath(req.file.path).then((response) => {
+        // save the image dimensions in new property for file
+        req.file.dimensions = response;
```

Comment:
I think i left this comment already? Why do we assign this to the file? 🤔 
Was there a reason? Let me know :)

Reply:
Yeah we talked about this on last PR :) The only reason it was done like this is because (blog-icon validation)[https://github.com/TryGhost/Ghost/blob/master/core/server/web/shared/middlewares/validation/blog-icon.js#L23] was already following this pattern of assigning dimensions to req. It doesn't seem to be getting used down the line for both, but wasn't sure if there was a reason for this or we need to fix in blog-icon as well :) If no reason there, then I can update in both places

Reply:
Thanks :) Yeah probably remove it? 🤔 It's weird, because then somebody will think it's used somewhere :D

Reply:
Or let's leave it as is, it's actually a correct behaviour, just not used.